modules/_labs/header/react/README.md

```diff
@@ -169,7 +169,7 @@ Default: `DubLogoTitle` (for "Dub" variants) or `WorkdayLogoTitle` (for "Full" v
 
 ---
 
-#### `onMenuClick: (React.SyntheticEvent) => void`
+#### `onMenuClick: (React.MouseEvent<HTMLButtonElement>) => void`
```

Comment:
Interesting side-note - a button click could be triggered by the "enter" or "space" keys. They are synthesized as a mouse click event with zeroed position information.